### PR TITLE
Feat: MsgInstallBundle indexing

### PR DESCRIPTION
**Pull request by frazarshad**
This PR adds indexing for `MsgInstallBundle` messages
image

Compare data in picture against this [transaction](https://ping.pub/agoric/tx/19298027C029ED9DEC1CE671BD2B89174220FE31A6F1BCDCEEB127ADA223EC20)

### CODE REVIEW

File: src/mappings/mappingHandlers.ts [hunk 1]
```diff
@@ -128,6 +129,29 @@ export async function handleIbcReceivePacketEvent(cosmosEvent: CosmosEvent): Pro
...
+    block.header.time as any,
+    BigInt(uncompressedSize),
+    bundle || '',
+    compressedBundle || '',
+    getAddressFromUint8Array(msg.decodedMsg.submitter),
+  );
+
+  await bundleRecord.save();
+}
+
```
Comment (rabi-siddique) [inline src/mappings/mappingHandlers.ts:154]: I think we should wrap this section of the code in a `try-catch` block to catch and log any errors that may occur.

File: src/mappings/utils.ts [hunk 2]
```diff
@@ -101,10 +101,15 @@ export function dateToDayKey(timestamp: any): number {
...
-  const chainPrefix = 'agoric';
   const stringtoSha = Buffer.from([...Buffer.from(version), 0, ...Buffer.from(`${port}/${channel}`)]);
   const shaHash = sha256.sha256.array(stringtoSha.toString());
-  const bechWords = bech32.toWords(shaHash.slice(0, 20));
-  const address = bech32.encode(chainPrefix, bechWords);
+  const address = getAddressFromUint8Array(shaHash.slice(0, 20));
   return address;
 };
+
+export const getAddressFromUint8Array = (uint8Array: Array<number>, chainPrefix: string = 'agoric') => {
```
Comment (rabi-siddique) [inline src/mappings/utils.ts:110]: You can use the `Uint8Array` type, 

```
export const getAddressFromUint8Array = (uint8Array: Uint8Array, chainPrefix: string = 'agoric') => {

```

Comment (rabi-siddique) [reply src/mappings/utils.ts:110]: If you change the type, you will need to make this change on line 106:

```js
  const shaHashUint8Array = new Uint8Array(shaHash.slice(0, 20));
  const address = getAddressFromUint8Array(shaHashUint8Array);
```

Comment (rabi-siddique) [reply src/mappings/utils.ts:110]: PS: I haven't tested it. Just checked in the code and there was no compilation errors.

**Review by rabi-siddique — APPROVED** [review]

File: src/mappings/mappingHandlers.ts (hunk 1 above)
Comment (frazarshad) [reply src/mappings/mappingHandlers.ts:154]: catching the error would result in skipping this message entirely. Throwing an error allows the indexing to stop at that point.
that is our current implementation for all of our functions

**Pull request merged**